chain-tx-enclave-next/enclave-ra/ra-client/src/config.rs

```diff
@@ -19,7 +19,8 @@ impl<'a> EnclaveCertVerifierConfig<'a> {
     pub fn new() -> Self {
         Self {
             signing_ca_cert_pem: IAS_CERT.into(),
-            valid_enclave_quote_statuses: vec!["OK".into()].into(),
+            // FIXME remove SW_HARDENING_NEEDED after https://github.com/rust-lang/llvm-project/pull/58
```

Comment:
I think `SW_HARDENING_NEEDED` will still appear as a response, so it may not be possible to remove it ("OK" or "UP-TO-DATE" will only appear on unaffected platforms which are only some very recent CPUs, i.e. mid-2019 and newer) -- it's more of a reminder that one should apply those software hardening measures: https://software.intel.com/security-software-guidance/insights/deep-dive-load-value-injection#mitigationguidelines

Since it affects many platforms, applying the mitigations and releasing only one version with them for all platforms is ok.

Reply:
Oh, then OK, UP-TO-DATE, SW_HARDENING_NEEDED all should be considered as valid responses?

Reply:
The API docs (https://api.trustedservices.intel.com/documents/sgx-attestation-api-spec.pdf) don't mention "UP-TO-DATE" (it's only mentioned in that LVI mitigation article), so perhaps only "OK" and "SW_HARDENING_NEEDED"

Reply:
Done